app/javascript/react/components/ParkTile.js

```diff
@@ -1,17 +1,19 @@
 import React from "react"
 
 const ParkTile = props => {
-
   return (
-    <div>
-      <h2>{props.name}</h2>
-      <h3 id="city">{props.city}</h3>
-      <h3 id="state">{props.state}</h3>
-      <h3 id="zip">{props.zip}</h3>
-      <h3 id="rating">{props.rating}</h3>
-      <img src={props.photo}/>
+    <div className="row" align="center">
+      <div className="columns small-10">
+        <div className="image-wrapper overlay-fade-in">
+          <img className="photo" src={props.photo}/>
+            <div className="image-overlay-content">
+              <h2 id="name">{props.name}</h2>
+              <p id="location">{props.city}, {props.state}</p>
+              <p id="rating">{props.rating} stars</p>
+            </div>
+        </div>
+      </div>
     </div>
   )
 }
-
 export default ParkTile
```

Comment:
add the empty lines back on 4 and what was 16 (now 18/19) to help with readability.